.travis/build.sh

```diff
@@ -78,9 +113,13 @@ RESULT=$(./wsk -i action invoke --blocking hello | grep "\"status\": \"success\"
 
 if [ -z "$RESULT" ]; then
   echo "FAILED! Could not invoked custom action"
+
+
+  echo " ----------------------------- controller logs ---------------------------"
+  kubectl -n openwhisk logs $(kubectl get pods --all-namespaces -o wide | grep controller | awk '{print $2}')
+  echo " ----------------------------- invoker logs ---------------------------"
+  kubectl -n openwhisk logs $(kubectl get pods --all-namespaces -o wide | grep invoker | awk '{print $2}')
```

Comment:
Nice improvements to provide instant log/trace of where a failure bw/ controller-invoker may have occurred.